app/services/mapping/declaration/AgentBuilder.scala

```diff
@@ -28,14 +28,11 @@ object AgentBuilder {
   def build(implicit cacheMap: CacheMap): Declaration.Agent =
     cacheMap
       .getEntry[RepresentativeDetails](RepresentativeDetails.formId)
-      .filter(isDefined)
+      .filter(RepresentativeDetails.isDefined)
```

Comment:
`.filter(RepresentativeDetails.isDefined)`
I would have done it
`filter(_.isDefined)` instead, as it is more functional, but that is ok with me.